webkit2/webkit2.settings.lisp

```diff
@@ -36,7 +36,6 @@
     ("enable-offline-web-application-cache" "gboolean" t t)
     ("enable-page-cache" "gboolean" t t)
     ("enable-plugins" "gboolean" t t)
-    ("enable-private-browsing" "gboolean" t t)
```

Comment:
Don't remove it as it may break dependent code.  Instead add a comment (unless you find a way to add a docstring).